app/main.py

```diff
@@ -1,3 +1,5 @@
+from app.knight import Knight
+
```

Comment:
2 blank lines should be added after the last import

Reply:
For some reason, Ctrl + Alt + L leaves 1 empty space here